docs/source/user_documentation/user_documentation.rst

```diff
@@ -125,6 +125,25 @@ SnowShu will report details of the created replica once completed.
 
 .. image:: /../assets/completed_replica.png 
 
+Creating An Incremental Replica
+------------------
+
+There's a simple way to rebuild an existing replica in case the ``replica.yml`` file has been changed. Instead of building a brand new replica, you can apply changes to the existing one.
+Incremental replica creates relations and loads data only for new entries found in ``replica.yml`` file, which are not already present in target replica image.
+The target for the incremental replica is actually a docker image name, which you can find by typing: 
+
+>>> docker images
+
+It usually starts with ``snowshu_replica_``. So for a replica named *hamburger-sandwich*:
+
+>>> snowshu create -i snowshu_replica_hamburger-sandwich
+
+or
+
+>>> snowshu create --incremental snowshu_replica_hamburger-sandwich
+
+Once completed you'll get a report with details of the replica with updated relations. 
```

Comment:
Again: this sentence sounds like the main purpose of incremental option is generating report instead of updating replica image. Check previous comment